portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACUtils.path

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+<title>ACUtils</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ACUtils</td></tr>
+</thead>
+<tbody>
+<!-- CLOSE MODAL -->
+<tr>
+    <td>MODAL_CLOSE_BUTTON</td>
+    <td>//button[contains(@class, 'close')]</td>
+    <td></td>
+</tr>
+<!-- SEARCH_BAR -->
+<tr>
+    <td>CLEAR_SEARCH_BAR_BUTTON</td>
+    <td>//div[contains(@class, 'input-group-item')]//button[contains(@class, 'button-root btn btn-unstyled')]</td>
```

Comment:
hi @r4ddek , when using contains, try to not use multiple classes/elements to not let your selector fragile. In this case, you can just have in the second part of your selector:
`//button[contains(@class, 'button-root')]`

Reply:
Ops, this was a leftover from a revert, thanks!!